taxonomy-editor-frontend/src/pages/editentry/ListAllOtherProperties.jsx

```diff
@@ -1,22 +1,97 @@
-import { Box, Typography } from "@mui/material";
+import { Box, Paper, Stack, TextField, Typography } from "@mui/material";
+import ISO6391 from 'iso-639-1';
 
 // Parent component used for rendering info
 // on a stopword, synonym, header or footer
```

Comment:
nit: if you use the other type of comment: `/** ...your comment here */` it is recognized as a doc string, please replace it

Reply:
what would be shown for header and footer? don't see anything related to those in the code... if not yet done please add a TODO, or remove it from the description